internal/orchestrator/orchestrator_test.go

```diff
@@ -454,6 +459,12 @@ func TestInitDataComponentFail(t *testing.T) {
 	assert.Regexp(t, "FF10128", err)
 }
 
+func TestInitMetricsComponentFailGracefully(t *testing.T) {
+	or := newTestOrchestrator()
+	or.metrics = nil
+	or.initComponents(context.Background())
```

Comment:
Maybe should check for `NoError` on this return?

Reply:
The title of the test seems wrong too. This isn't a failure test right? There is no way metrics init can fail. It just succeeds.